sklearn_pandas/dataframe_mapper.py

```diff
@@ -206,7 +208,7 @@ def transform(self, X):
             if self.default is not None:
                 Xt = self.default.transform(Xt)
             extracted.append(_handle_feature(Xt))
-
+            self.transformed_names_ += self.get_names(unsel_cols, self.default, Xt)
```

Comment:
Not sure why originally there was no index = index + self._get_names(.... over here.